sources/packages/web/src/views/institution/EditInstitutionLocation.vue

```diff
@@ -38,20 +37,22 @@ export default {
   setup(props: any) {
     // Hooks
     const store = useStore();
-    const initialData = ref({});
+    const initialData = ref({} as InstitutionLocationEdit);
     const toast = useToast();
     const router = useRouter();
-    const submitted = async (data: InstitutionLocationFormAPIInDTO) => {
+    const updateInstitutionLocation = async (
+      data: InstitutionLocationPrimaryContactAPIInDTO,
+    ) => {
       try {
         await InstitutionService.shared.updateInstitutionLocation(
           props.locationId,
           data,
         );
-        router.push({ name: InstitutionRoutesConst.MANAGE_LOCATIONS });
+        router.push(goBackRouteParams.value);
         store.dispatch("institution/getUserInstitutionLocationDetails");
         toast.add({
           severity: "success",
-          summary: `Your location information for ${data.locationName} have been updated`,
+          summary: `Your location information have been updated`,
```

Comment:
maybe you can replace this too with `useToastMessage`

Reply:
@ann-aot there is no "maybe" here, it is not open for discussion 😉 
`useToastMessage` must be used.

Reply:
i said maybe becuase, I think this was not part of @guru-aot  PR. i appreciate if its replaced with `useToastMessage`